CONTRIBUTING.md

```diff
@@ -0,0 +1,294 @@
+[GitHub issue](https://github.com/YaleSTC/vesta/issues/new).
+Before doing so, however, please **search open / closed issues** to see if
+someone else has already opened a similar issue.
+
+### Bug Reports
+A bug is a *verifiable problem* that is caused by the Vesta code. Good
+bug reports allow us to quickly identify the root causes of problems and
+resolve them. Here are some guidelines for submitting bug reports:
+
+1. **Is it a duplicate?** - check to see if the bug has already been reported.
+2. **Has it been resolved?** - check if it can be reproduced using the `master`
+   branch or a newer version.
+3. **What is the problem?** - see if you can isolate exactly how to reproduce
+   the issue.
+4. **Include a screenshot or screencast** - you can use [LICEcap](http://www.cockos.com/licecap/)
+   to capture a screencast and save it as an animated gif.
+5. **Tell us all the things** - include as much information as possible (see
+   the template below for a good starting point).
+
+#### Bug Report Template (thanks [Ghost](https://github.com/TryGhost/Ghost) team!)
+```
+Short and descriptive example bug report title
+
+### Issue Summary
+
+A summary of the issue and the browser/OS environment in which it occurs. If
+suitable, include the steps required to reproduce the bug.
+
+### Steps to Reproduce
+
+1. This is the first step
+2. This is the second step
+3. Further steps, etc.
+
+Any other information you want to share that is relevant to the issue being
+reported. Especially, why do you consider this to be a bug? What do you expect
+to happen instead?
+
+### Technical details:
+
+* Vesta Version: [vX.X.X OR master (a1b2c3)]
+* Operating System: [Operating System X]
+* Browser: [Browser vX.X.X]
+```
+
+A member of the core team will label your issue appropriately and may follow
+up for more information. We try to deal with bugs as quickly as possible and
+will keep the issue updated with our progress, but it may take some time
+before it is resolved.
+
+### Feature Requests
+Feature requests are always welcome as well. Before submitting a feature
+request, please search both opened and closed issues to see if the feature has
+already been requested. Remember that it's up to *you* to convince the core
+team of the merits of the feature. Please include as much detail as possible,
+including but not limited to the use case, a proposed implementation, and why
+it is likely to be useful.
+
+## Improving Documentation
+One great way of helping out is improving documentation, which can take the
+form of user-, developer- , and sysadmin-facing documentation. Our user-facing
+documentation is located on our GitHub Pages [site](https://yalestc.github.io/vesta),
+while our developer- and sysadmin-facing documentation is located in code
+comments, in our [README](https://github.com/YaleSTC/vesta/blob/master/README.md),
+in the `/doc` folder of the repo, and in our [GitHub wiki](https://github.com/YaleSTC/vesta/wiki).
```

Comment:
should we maybe take this out and add it back when we actually have this documentation?